<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pt::MainWindow</class>
 <widget class="QMainWindow" name="pt::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="pt::TorrentListWidget" name="torrentList"/>
      <widget class="pt::TorrentDetailsWidget" name="torrentDetails">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionAddTorrent"/>
    <addaction name="actionAddMagnetLink"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>filter</string>
     </property>
     <addaction name="actionFilterNone"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuFilter"/>
    <addaction name="separator"/>
    <addaction name="actionDetailsPanel"/>
    <addaction name="actionStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="pt::StatusBar" name="statusBar"/>
  <action name="actionAddTorrent">
   <property name="text">
    <string>add_torrent_s</string>
   </property>
  </action>
  <action name="actionAddMagnetLink">
   <property name="text">
    <string>add_magnet_link_s</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionDetailsPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>details_panel</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>status_bar</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>preferences</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>check_for_update</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionFilterNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pt::TorrentListWidget</class>
   <extends>QTreeView</extends>
   <header>torrentlistwidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>pt::TorrentDetailsWidget</class>
   <extends>QTabWidget</extends>
   <header>torrentdetails/torrentdetailswidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pt::StatusBar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
